test/firedrake/test_implicit_friction.py

```diff
@@ -35,14 +36,14 @@ def test_implicit_friction(do_export=False, do_assert=True):
 
     # ----- define function spaces
     deg = 1
-    p1dg = FunctionSpace(mesh, 'DG', degree=1, vfamily='DG', vdegree=1)
-    p1dgv = VectorFunctionSpace(mesh, 'DG', degree=1, vfamily='DG', vdegree=1)
-    u_h_elt = FiniteElement('RT', triangle, deg + 1)
-    u_v_elt = FiniteElement('DG', interval, deg)
+    p1dg = get_functionspace(mesh, 'DG', 1)
+    p1dgv = get_functionspace(mesh, 'DG', 1, vector=True)
+    u_h_elt = FiniteElement('RT', triangle, deg + 1, variant='equispaced')
```

Comment:
Note that this doesn't actually do anything since we don't have spectral on triangles.

Reply:
Yeah, true.